mediabridge/integrations/description_generator_temp.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+import json
+
+from mediabridge.definitions import PROJECT_DIR
+from mediabridge.integrations.ollama_api import generate_prompt_response
+
+"""
+Program that reads movie_titles.txt from mediabridge/data and creates a JSONL file with
+the movie title, year, and a description generated by the Ollama API.
+
+Run with:
+    PYTHONPATH=$(pwd) python mediabridge/integrations/description_generator_temp.py --skip-lines 208
+"""
+
+
+def generate_descriptions(skip_lines: int = 0):
+    ADJ_DATA_DIR = PROJECT_DIR / "mediabridge" / "data"
+    input_path = ADJ_DATA_DIR / "movie_titles.txt"
+    output_path = ADJ_DATA_DIR / "movie_titles_plus_descriptions.jsonl"
+
+    with open(input_path, "r", encoding="latin1") as f1:
+        for i, raw_line in enumerate(f1):
+            if i < skip_lines:
+                continue
+
+            line = raw_line.strip().split(",", maxsplit=2)
+            if len(line) < 3:
+                continue  # skip bad lines
+
+            movie_id, year, title = line
+
+            print(f"writing description for {title}...")
+
+            prompt = f"Write a paragraph description for the {year} production '{title}' — no introductions, titles, or extra formatting."
+            response = generate_prompt_response(prompt=prompt)
+
+            movie_data = {
+                "id": movie_id,
+                "year": year,
+                "title": title,
+                "description": response.strip(),
+            }
+
+            with open(output_path, "a", encoding="utf-8") as f2:
```

Comment:
Not for this PR, but another "architecture" disccusion. We should store data in a database. A proliferation of .csv and .jsonl files will make this project harder and harder to understand, manage and improve upon. Instead, we should strive for a centralized database and schema, so that we're all on the "same page" and can effectively build upon each other's work. I understand that this is maybe the opposite of my encouragement a couple of weeks ago to "just build whatever you weant", but we should definitely keep it in mind.